lib/config/definitions.ts

```diff
@@ -1695,6 +1695,16 @@ const options: RenovateOptions[] = [
     cli: false,
     env: false,
   },
+  {
+    name: 'authType',
+    description:
+      'Authentication type for http header. e.g. "Bearer" or "Basic".',
```

Comment:
Is there a need to make this freeform or should we limit it to `Bearer` and `Basic`? Also can we default it to `Bearer` for clarity?

Reply:
this can be any string value from http standard. so if we leave it open we are future proof.